### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation

### DISCUSSION THREAD

**Discussion by raravena80** [projectcapsule/capsule#670] [General] [answered]
Hello Capsule team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I believe you had an agenda item to present the project in our meeting. For example, it would be great to learn things like the roadmap, tech, community, general overview of the project.

Feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com) at your convenience. 

Thanks!

**Comment by prometherion** [comment, marked as answer]
Thanks, Ricardo! Looking forward to it!

I'll start an email thread to find the most suitable date according to all the maintainers.
